nilearn/_utils/path_finding.py

```diff
@@ -10,10 +10,12 @@ def resolve_globbing(path):
     """Resolve globbing patterns in a path."""
     path = stringify_path(path)
     if isinstance(path, str):
-        path_list = sorted(glob.glob(str(Path(path).expanduser())))
+        expanded_path = Path(path).expanduser()
+        path_list = sorted(glob.glob(str(expanded_path)))
```

Comment:
Will Ruff complain about the use of `glob.glob` instead of `pathlib.Path.glob`/`pathlib.Path.rglob`?